src/libstore/store-api.hh

```diff
@@ -115,6 +115,7 @@ struct ValidPathInfo
 {
     StorePath path;
     std::optional<StorePath> deriver;
+    std::optional<string> outputname;
```

Comment:
Is the intent that we can have `outputname` whenever we have `deriver`, except for backwards compat with old infos?

Reply:
BTW it looks like this field is not actually used anywhere.

Reply:
Indeed, I've added it for an experiment and forgot to remove it. This field doesn't really make any sense (even the `deriver` field should probably be deprecated as a store path can now have several derivers)

Reply:
Yeah once we upload the trust mapping, we'll probably want to key it by the derivation not the output path, which also helps different signatures sign for different derivers.